docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,378 @@
+      "text/plain": [
+       "<Figure size 1400x480 with 1 Axes>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Draw timeline of circuit with fractional gates\n",
+    "draw_timeline(\n",
+    "    ising_circuit_fractional,\n",
+    "    idle_wires=False,\n",
+    "    time_range=(0, 500),\n",
+    "    style=IQXSimple(),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48bad5e6-3c7a-4bd0-b050-e440c3257be2",
+   "metadata": {},
+   "source": [
+    "## Where to use fractional gates\n",
+    "\n",
+    "Historically, the basis gates available on IBM Quantum QPUs have been **`CZ`**, **`X`**, **`RZ`**, **`SX`**, and **`ID`**, which can not efficiently represent circuits with single and two-qubit rotations that are not multiples of $\\pi / 2$. For example, an $R_X(\\theta)$ gate, when transpiled, must decompose into a series of $RZ$ and $\\sqrt{X}$ gates -- creating a circuit with two gates of finite duration instead of one.\n",
```

Comment:
```suggestion
    "Historically, the basis gates available on IBM Quantum QPUs have been **`CZ`**, **`X`**, **`RZ`**, **`SX`**, and **`ID`**, which can not efficiently represent circuits with single- and two-qubit rotations that are not multiples of $\\pi / 2$. For example, an $R_X(\\theta)$ gate, when transpiled, must decompose into a series of $RZ$ and $\\sqrt{X}$ gates, which creates a circuit with two gates of finite duration instead of one.\n",
```